rc-events-service/src/main/scala/com/epam/random_coffee/events/repo/EventRepository.scala

```diff
@@ -0,0 +1,15 @@
+package com.epam.random_coffee.events.repo
+
+import com.epam.random_coffee.events.model.{ Event, EventId }
+
+import scala.concurrent.Future
+
+trait EventRepository {
+  def save(event: Event): Future[Unit]
+
+  def find(id: EventId): Future[Option[Event]]
```

Comment:
[OPT]
I would name it like `get` in case when we really will want to use `find` with such a contract
`def find(filter: EventFilter, order: Order): Future[Seq[Event]]` or something like that

Reply:
changed